openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/widgets/CanvasimagevalueWidget.vue

```diff
@@ -56,44 +56,43 @@ export default {
     return {
       images: [],
       defaultImage: null,
-      screen: null,
       screenTarget: null,
       screenName: null,
     }
   },
   computed: {
-    valueId: function () {
-      return `${this.parameters[0]}__${this.parameters[1]}__${
-        this.parameters[2]
-      }__${this.parameters[3] || 'RAW'}`
-    },
     selectedValue: function () {
-      return this.$store.state.tlmViewerValues[this.valueId][0]
+      if (this.screen) {
+        if (this.screen.screenValues[this.valueId]) {
+          return this.screen.screenValues[this.valueId][0]
+        }
+      } else if (this.$store.state.tlmViewerValues[this.valueId]) {
+        return this.$store.state.tlmViewerValues[this.valueId][0]
+      }
+      return null
```

Comment:
Don't all widgets have a screen? When is the else case used?

Reply:
We can probably eliminate the $store values but that will be a large refactor.  Let's talk tomorrow.

Reply:
Eleminated $store and opened a new ticket for further refactoring